startup.sql

```diff
@@ -0,0 +1,41 @@
+/* This script creates the database for capurace. */
+/* The description of the database is in the wiki, Database-Structure.md */
+
+CREATE DATABASE IF NOT EXISTS capurace;
+USE capurace;
+
+CREATE TABLE IF NOT EXISTS user (
+        school VARCHAR(30) NOT NULL DEFAULT NULL UNIQUE KEY,
+        id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        leader VARCHAR(10) NOT NULL DEFAULT NULL,
+        tel VARCHAR(11) NOT NULL DEFAULT NULL,
+        mail VARCHAR(30) NOT NULL DEFAULT NULL,
+        password VARCHAR(30) NOT NULL DEFAULT NULL,
+        nteams SMALLINT UNSIGNED NOT NULL DEFAULT 0,
+        bill INT NOT NULL DEFAULT 0,
+        paid SMALLINT NOT NULL DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS people (
+        name VARCHAR(10) NOT NULL DEFAULT NULL,
+        id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        gender INT NOT NULL DEFAULT NULL,
+        idcard VARCHAR(40) NOT NULL DEFAULT NULL,
+        schoolid INT NOT NULL DEFAULT NULL,
+        accommodation INT NOT NULL DEFAULT 1,
+        meal INT NOT NULL DEFAULT 1,
+        race INT NOT NULL DEFAULT NULL,
+        teamrace INT NOT NULL DEFAULT NULL
```

Comment:
What about just store the team id here? (We need to change the description in wiki.